cwl/src/main/scala/cwl/Workflow.scala

```diff
@@ -27,9 +27,20 @@ case class Workflow private(
                      outputs: Array[WorkflowOutputParameter],
                      steps: Array[WorkflowStep],
                      requirements: Option[Array[Requirement]],
-                     hints: Option[Array[CwlAny]]) {
-  def womExecutable(inputFile: Option[String] = None): Checked[Executable] = {
-    CwlExecutableValidation.buildWomExecutable(womDefinition, inputFile)
+                     hints: Option[Array[Hint]]) {
+  /** Builds an `Executable` from a `Workflow` CWL with no parent `Workflow` */
+  def womExecutable(validator: RequirementsValidator, inputFile: Option[String] = None): Checked[Executable] = {
+    CwlExecutableValidation.buildWomExecutable(womDefinition(validator), inputFile)
+  }
+
+  var parentWorkflow: Option[Workflow] = None
```

Comment:
This should probably be `private` or `private[cwl]`

Reply:
Also I don't see it being set anywhere ?

Reply:
It's not being set currently, the subworkflow stuff hadn't landed when I started this.